inspirehep/modules/forms/validation_utils.py

```diff
@@ -1,172 +1,39 @@
-                    )
-                )
-
-
-class RequiredIf(object):
-
-    """Require field if value of another field is set to a certain value."""
-
-    def __init__(self, other_field_name, values, message=None):
-        """Initialize the validator."""
-        self.other_field_name = other_field_name
-        self.values = values
-        self.message = message
-
-    def __call__(self, form, field):
-        """Validate."""
-        try:
-            other_field = getattr(form, self.other_field_name)
-            other_val = other_field.data
-            for v in self.values:
-                # Check if field value is required
-                if (callable(v) and v(other_val)) or (other_val == v):
-                    # Field value is required - check the value
-                    if not field.data or \
-                            isinstance(field.data, six.string_types) \
-                            and not field.data.strip():
-                        if self.message is None:
-                            self.message = 'This field is required.'
-                        field.errors[:] = []
-                        raise StopValidation(self.message % {
-                            'other_field': other_field.label.text,
-                            'value': other_val
-                        })
-        except AttributeError:
-            pass
-
-
-class NotRequiredIf(RequiredIf):
-
-    """Do not require field if another field contains a certain value."""
-
-    def __call__(self, form, field):
-        """Validate."""
-        try:
-            other_field = getattr(form, self.other_field_name)
-            other_val = other_field.data
-            for v in self.values:
-                # Check if field value is not required.
-                if (callable(v) and v(other_val)) or (other_val == v):
-                    raise StopValidation()
-        except AttributeError:
-            pass
-
-
-class Unchangeable(object):
-
-    """Unchangeable validator."""
-
-    def __call__(self, form, field):
-        """Validate."""
-        field.data = field.object_data
-
-
-def number_validate(form, field, submit=False,
-                    error_message='It must be a number!'):
-    """Number validator."""
-    value = field.data or ''
-    if value == "" or value.isspace():
-        return
-
-    def is_number(s):
-        """Decide whether argument is a number."""
-        try:
-            float(s)
-            return True
-        except ValueError:
-            return False
-
-    if not is_number(value):
-        raise ValidationError(error_message)
-
-#
-# DOI-related validators
-#
-
-
-def doi_syntax_validator(form, field):
-    """DOI syntax validator. Deprecated.
-
-    :param field: validated field.
-    :param form: validated form.
-    """
-    import warnings
-    warnings.warn("Please use DOISyntaxValidator instead.", DeprecationWarning)
-    return DOISyntaxValidator()(form, field)
```

Comment:
`doi_syntax_validator` has no uses:
```shell
% ag doi_syntax_validator
inspirehep/modules/forms/validation_utils.py
161:def doi_syntax_validator(form, field):
```
while `DOISyntaxValidator` has several:
```shell
% ag DOISyntaxValidator
inspirehep/modules/forms/fields/doi.py
26:from ..validation_utils import DOISyntaxValidator
51:                DOISyntaxValidator(),

inspirehep/modules/forms/validation_utils.py
168:    warnings.warn("Please use DOISyntaxValidator instead.", DeprecationWarning)
169:    return DOISyntaxValidator()(form, field)
172:class DOISyntaxValidator(object):

inspirehep/modules/literaturesuggest/forms.py
44:from inspirehep.modules.forms.validation_utils import DOISyntaxValidator
234:        validators=[DOISyntaxValidator("The provided DOI is invalid - it should look similar to '10.1086/305772'."),
```
since it was already deprecated, why don't we remove it?